doc/release-notes/rl-2511.section.md

```diff
@@ -16,6 +16,16 @@
   If a newer C++ library feature is not available on the default deployment target, you will need to increase the deployment target.
   See the Darwin platform documentation for more details.
 
+- A great number of changes to CUDA packaging:
+  - CUDA 13 is now available.
+  - Creation of CUDA package sets has been rewritten to remove module system evaluation.
+  - More packages provided by NVIDIA as binary archives (also called redistributables) are now available.
+  - Packages which fail to evaluate and depend on CUDA packages can produce helpful diagnostic messages during evaluation when `--show-trace --trace-verbose` are used.
+  - Package expressions are being updated to support building against older releases (where appropriate) to support out-of-tree consumers like [cuda-legacy](https://github.com/nixos-cuda/cuda-legacy).
+  - `_cuda.lib.licenses` was created to introduce several NVIDIA licenses new to Nixpkgs and used almost entirely within the CUDA package set.
+  - `_cuda.lib.getRedistSystem` had an API change to accommodate a new dependency on CUDA version.
+  - Many members of the CUDA package set are now marked as broken if `config.cudaSupport` is not set. This change was made to prevent the common mistake of using CUDA tooling to build a CUDA application but forgetting to enable `config.cudaSupport`, resulting in an application which supports CUDA and dependencies which do not.
```

Comment:
Disclaimer about the breaking change, and a reference to the config option for recovering the unsafe behaviour

Reply:
> not introducing a way to re-arm the foot gun (at least not now, and not in this PR).

OK I open a separate one

Reply:
I'm thinking... in principle, the unsafe behaviour can be achieved OOT using overlays instead of the extra config 🤔